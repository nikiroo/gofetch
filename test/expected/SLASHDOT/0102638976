    GOOGLE INVESTIGATING ISSUE WITH BLURRY FONTS ON NEW CHROME 69  
                             (ZDNET.COM)                           

  Thursday September 06, 2018 @11:30PM (msmash)
  from the blurry-texts dept.

  o Reference: 0102638976
  o News link: https://tech.slashdot.org/story/18/09/06/1839242/google-investigating-issue-with-blurry-fonts-on-new-chrome-69
  o Source link: https://www.zdnet.com/article/google-investigating-issue-with-blurry-fonts-on-new-chrome-69/


    Since  the release of Chrome 69 earlier this week, countless of
    users  have  gone  on social media and Google Product Forums to
    complain  about  "blurry" or "fuzzy" text inside Chrome. ZDNet:
    The  blurred  font  issue  isn't  only limited to text rendered
    inside   a  web  page,  users  said,  but  also  for  the  text
    suggestions  displayed inside the address bar search drop-down,
    and   Chrome's   Developer  Tools  panel.  [...]  According  to
    reports,  the  issue  only  manifests  for  Chrome  69 users on
    Windows.  Those  who  rolled  back  to Chrome 68 stopped having
    problems.  Users  said  that changing Chrome, operating system,
    or  screen DPI settings didn't help. "Our team is investigating
    reports  of  this  behavior.  You  can find more information in
    this  public bug report," a Google spokesperson said last night
    after  first  user  complaints  started  surfacing online. Some
    users  have  also  expressed  concerns  over Chrome not showing
    "trivial  subdomains" including www and secure lock sign in the
    address bar.


  ** Whats the deal (Score:2)
     (by 110010001000 ( 697113 ))

     
     Whats the deal with the new interface. Looks like we are back to
     rounded corners again.

     ** 

        ** Re: (Score:1)
           (by 110010001000 ( 697113 ))

           
           And what's the deal with decaf coffee? How do they get the
           caffeine out of there, and then where does it go?

           ** Re: (Score:1)
              (by Anonymous Coward)

              
              Why do they put caffeine in coffee and then take it
              back out?
              And why if Chrome rendering fonts in the first place?
              That's a job for the operating system.


           ** Re: Whats the deal (Score:2)
              (by spongman ( 182339 ))

              
              dichloromethe (or ethyl acetate) leeching.



        ** Re: (Score:2)
           (by DontBeAMoran ( 4843879 ))

           
           Score: 5, That's gold Jerry, GOLD!



  ** So Chrome 69 makes you blind? (Score:4, Funny)
     (by the_skywise ( 189793 ))

     
     The jokes write themselves...

  ** Blurry Fonts... (Score:2)
     (by Oswald McWeany ( 2428506 ))

     
     Maybe they need to put on their Google Glass so they can read
     the fonts clearer.

     ** Re: (Score:3)
        (by Z00L00K ( 682162 ))

        
        They just started to use Cleartype and the fuzzy fonts.
        Both cleartype and fuzzy fonts gives me a headache so I have
        to use uBlock to revert to the browser default when I run
        Firefox.


  ** I bet you (Score:5, Insightful)
     (by The MAZZTer ( 911996 ))

     
     If these are Windows users they screwed with the Compatibility
     Mode options for DPI in order to make the window smaller or
     bigger. It looked fine until Google did something differently
     and now the Compatibility Mode options make it look blurry.
     Turns out using options designed for use on legacy applications
     only on modern applications introduces problems. Lots of Steam
     users have similar problems.

     ** Re: (Score:2)
        (by omnichad ( 1198475 ))

        
        Wrong. I use the standard DPI settings for the whole screen
        (1.5x on my 27" 4K screen), which only works with programs
        that are DPI-aware. It's true that I didn't notice the blurry
        fonts until I saw the headline, but I really just thought I
        was that tired (I've only had 69 for about a day). At least
        for me, it's fairly mild.

        ** 

           ** Re: (Score:2)
              (by omnichad ( 1198475 ))

              
              If it looks significantly worse, it could also have
              nothing at all to do with the bug - Chrome is DPI aware
              and should never have that enabled. Especially since
              the bug report doesn't have any mention of anything but
              the system DPI setting.



        ** 

           ** Re: (Score:2)
              (by omnichad ( 1198475 ))

              
              So you like the way I word it (No Giggity).




     ** Re: (Score:3)
        (by Solandri ( 704621 ))

        
        The problem is there are three ways to tweak the DPI.
        Compatibility Mode, which can be set on a per-app basis.
        Display scaling, which is set via the Windows desktop and
        affects everything. And Advanced Scaling which Microsoft
        introduced with the Spring Creators Update, which I haven't
        figured out exactly how it's different from the regular
        Display Scaling.
        
        Under the hood, there's also ClearType which runs on top of
        DPI scaling, and does [1]subpixel rendering  [grc.com] (using
        the individual red, green, and blue subp
        
        
        
        
        [1] https://www.grc.com/ctwhat.htm

        ** Re: (Score:2)
           (by omnichad ( 1198475 ))

           
           Looking at some zoomed in screenshots, I think they're
           scaling the subpixel rendering itself (which should never
           happen). There is no reason to have red or blue tint on
           anything but the outermost pixel, but looking at my
           screenshot close up I'm seeing two side-by-side bluish or
           reddish pixels.



     ** Re: (Score:2)
        (by Scoth ( 879800 ))

        
        Even on Chrome 69 on my multi-DPI work setup I have to turn
        on the DPI scaling compatibility mode/override to not get
        huge dialog boxes and widgets. It's a bit frustrating to me
        that a currently supported, modern app would still have
        problems with that.


     ** Re: (Score:2)
        (by Ingenium13 ( 162116 ))

        
        Nope, I have a Windows VM that's essentially stock that I use
        for a couple applications. I had Chrome open in it earlier
        today, and noticed that the fonts were blurry and were giving
        me a headache after a few minutes of use. I thought maybe it
        was something with RDP messing it up, but after I read this,
        I logged in again and verified that the fuzzy fonts are only
        in Chrome. All other applications are fine.


     ** Re: (Score:2)
        (by greenwow ( 3635575 ))

        
        And most Windows apps. For most of our users on Windows 10,
        Windows apps have blurry text no matter what settings we try.
        That really sucks for developers that need to read a lot of
        text.


     ** 

        ** Re: I bet you (Score:2)
           (by spongman ( 182339 ))

           
           They don't?



  ** Not new (Score:2)
     (by Tailhook ( 98486 ))

     
     Chrome has been having this problem for certain desktop users
     for a while now. There are workaround flags and whatnot. Just
     another chronic quality problem in Chrome that never gets
     fixed....

     ** Re: (Score:1)
        (by MidSpeck ( 1516577 ))

        
        True. I had to disable "Accelerated 2D canvas" in order to
        get the fuzziness to go away on mine on Chrome 68, which was
        working just fine before that.


  ** 

     ** Re: (Score:2)
        (by Z34107 ( 925136 ))

        
        Text wrapping is a "solved" problem, too, but you're still
        doing it by hand for some reason.


  ** While they're at it (Score:2)
     (by Pedestrianwolf ( 1591767 ))

     
     ..maybe they could also roll back all rounded rectangles they
     added in v69. Everything is so round it feels like I designed it
     in my basement.

     ** Re: (Score:3)
        (by ArchieBunker ( 132337 ))

        
        Chrome has a major UI re-design? Firefox is going to be
        pulling some late nights to catch up!


  ** It's also has performance regressions (Score:2)
     (by Suren Enfiajyan ( 4600031 ))

     
     On my Lenovo G570 the latest chrome also has very high CPU usage
     on [1]slither.io game [slither.io] and the game turns into a
     slideshow in places with many snakes. Firefox 61 doesn't suffer
     from the this problem, despite the fact that its slower almost
     in any regard. But not this time.
     
     
     
     
     [1] http://slither.io/

  ** BSOD (Score:2)
     (by jimbrooking ( 1909170 ))

     
     After installing ALL pages I visit are pure black. No text or
     graphics.

     ** Re: (Score:2)
        (by jimbrooking ( 1909170 ))

        
        Fix (Win 7 Home): Uninstall Chrome, delete all browsing date.
        reinstall Chrome, be dazzled by the curviness.


  ** Can someone summarize the linked article? (Score:4, Funny)
     (by kaizendojo ( 956951 ))

     
     I upgraded and I can't read it.

  ** See Jeff Probst ... (Score:2)
     (by CaptainDork ( 3678879 ))

     
     ... about this.
     I remember on SurvivorÂ®, back in the day, they had a "blurry
     tit," problem.
     They fixed that by disallowing exposed tits.
     Google should remove the tits from their fonts.

  ** trivial subdomain hiding (Score:2)
     (by Zaiff Urgulbunger ( 591514 ))

     
     So this "trivial subdomain hiding" thing... it's a crap idea
     right?

  ** Here's my take as a Mac user (Score:2)
     (by DontBeAMoran ( 4843879 ))

     
     First of all, I really hate these thin fonts. They may look neat
     on hi-DPI displays but not everyone has one.
     Secondly, the text on the left screen capture looks like it had
     its pixels hammered to the nearest pixel, which is the typical
     crappy-looking Microsoft anti-aliasing while the text on the
     right looks like normal anti-aliased text to me.


