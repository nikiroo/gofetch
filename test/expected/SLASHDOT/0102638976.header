0Google Investigating Issue With Blurry Fonts on new Chrome 69 (zdnet.com)	null/SLASHDOT/0102638976		70
i  Thursday September 06, 2018 @11:30PM (msmash)
i  from the blurry-texts dept.
i
i    Since  the release of Chrome 69 earlier this week, countless of
i    users  have  gone  on social media and Google Product Forums to
i    complain  about  "blurry" or "fuzzy" text inside Chrome. ZDNet:
i    The  blurred  font  issue  isn't  only limited to text rendered
i    inside   a  web  page,  users  said,  but  also  for  the  text
i    suggestions  displayed inside the address bar search drop-down,
i    and   Chrome's   Developer  Tools  panel.  [...]  According  to
i    reports,  the  issue  only  manifests  for  Chrome  69 users on
i    Windows.  Those  who  rolled  back  to Chrome 68 stopped having
i    problems.  Users  said  that changing Chrome, operating system,
i    or  screen DPI settings didn't help. "Our team is investigating
i    reports  of  this  behavior.  You  can find more information in
i    this  public bug report," a Google spokesperson said last night
i    after  first  user  complaints  started  surfacing online. Some
i    users  have  also  expressed  concerns  over Chrome not showing
i    "trivial  subdomains" including www and secure lock sign in the
i    address bar.
i
