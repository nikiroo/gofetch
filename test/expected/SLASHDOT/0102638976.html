<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>Google Investigating Issue With Blurry Fonts on new Chrome 69 (zdnet.com)</h1>
	<div class='details'>(Thursday September 06, 2018 @11:30PM (msmash) 
from the blurry-texts dept.)</div>
	<br/>
    <ul>
        <li>News link: <a href='https://tech.slashdot.org/story/18/09/06/1839242/google-investigating-issue-with-blurry-fonts-on-new-chrome-69'>https://tech.slashdot.org/story/18/09/06/1839242/google-investigating-issue-with-blurry-fonts-on-new-chrome-69</a></li>
        <li>Source link: <a href='https://www.zdnet.com/article/google-investigating-issue-with-blurry-fonts-on-new-chrome-69/'>https://www.zdnet.com/article/google-investigating-issue-with-blurry-fonts-on-new-chrome-69/</a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Since the release of Chrome 69 earlier this week, countless of users have gone on social media and Google Product Forums to complain about &quot;blurry&quot; or &quot;fuzzy&quot; text inside Chrome. ZDNet: The blurred font issue isn&#x27;t only limited to text rendered inside a web page, users said, but also for the text suggestions displayed inside the address bar search drop-down, and Chrome&#x27;s Developer Tools panel. [...] According to reports, the issue only manifests for Chrome 69 users on Windows. Those who rolled back to Chrome 68 stopped having problems. Users said that changing Chrome, operating system, or screen DPI settings didn&#x27;t help. &quot;Our team is investigating reports of this behavior. You can find more information in this public bug report,&quot; a Google spokesperson said last night after first user complaints started surfacing online. Some users have also expressed concerns over Chrome not showing &quot;trivial subdomains&quot; including www and secure lock sign in the address bar.
	</div>
<hr/>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>Whats the deal (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by 110010001000 ( 697113 )</div>
    <div class='comment_content'><p></p><p>Whats the deal with the new interface. Looks like we are back to rounded corners again.</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2></h2>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h2>Re: (Score:1)</h2>
        <div class='by' style='font-style: italic;'>by 110010001000 ( 697113 )</div>
        <div class='comment_content'><p></p><p>And what's the deal with decaf coffee? How do they get the caffeine out of there, and then where does it go?</p></div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h2>Re: (Score:1)</h2>
          <div class='by' style='font-style: italic;'>by Anonymous Coward</div>
          <div class='comment_content'><p></p><p>Why do they put caffeine in coffee and then take it back out?</p><p>And why if Chrome rendering fonts in the first place? That's a job for the operating system.</p></div>
        </div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h2>Re: Whats the deal (Score:2)</h2>
          <div class='by' style='font-style: italic;'>by spongman ( 182339 )</div>
          <div class='comment_content'><p></p><p>dichloromethe (or ethyl acetate) leeching.</p></div>
        </div>
      </div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h2>Re: (Score:2)</h2>
        <div class='by' style='font-style: italic;'>by DontBeAMoran ( 4843879 )</div>
        <div class='comment_content'><p></p><p>Score: 5, That's gold Jerry, GOLD!</p></div>
      </div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>So Chrome 69 makes you blind? (Score:4, Funny)</h2>
    <div class='by' style='font-style: italic;'>by the_skywise ( 189793 )</div>
    <div class='comment_content'><p></p><p>The jokes write themselves...</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>Blurry Fonts... (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by Oswald McWeany ( 2428506 )</div>
    <div class='comment_content'><p></p><p>Maybe they need to put on their Google Glass so they can read the fonts clearer.</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:3)</h2>
      <div class='by' style='font-style: italic;'>by Z00L00K ( 682162 )</div>
      <div class='comment_content'><p></p><p>They just started to use Cleartype and the fuzzy fonts.</p><p>Both cleartype and fuzzy fonts gives me a headache so I have to use uBlock to revert to the browser default when I run Firefox.</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>I bet you (Score:5, Insightful)</h2>
    <div class='by' style='font-style: italic;'>by The MAZZTer ( 911996 )</div>
    <div class='comment_content'><p></p><p>If these are Windows users they screwed with the Compatibility Mode options for DPI in order to make the window smaller or bigger. It looked fine until Google did something differently and now the Compatibility Mode options make it look blurry. Turns out using options designed for use on legacy applications only on modern applications introduces problems. Lots of Steam users have similar problems.</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:2)</h2>
      <div class='by' style='font-style: italic;'>by omnichad ( 1198475 )</div>
      <div class='comment_content'><p></p><p>Wrong. I use the standard DPI settings for the whole screen (1.5x on my 27" 4K screen), which only works with programs that are DPI-aware. It's true that I didn't notice the blurry fonts until I saw the headline, but I really just thought I was that tired (I've only had 69 for about a day). At least for me, it's fairly mild.</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h2></h2>
        <div class='by' style='font-style: italic;'></div>
        <div class='comment_content'></div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h2>Re: (Score:2)</h2>
          <div class='by' style='font-style: italic;'>by omnichad ( 1198475 )</div>
          <div class='comment_content'><p></p><p>If it looks significantly worse, it could also have nothing at all to do with the bug - Chrome is DPI aware and should never have that enabled. Especially since the bug report doesn't have any mention of anything but the system DPI setting.</p></div>
        </div>
      </div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h2></h2>
        <div class='by' style='font-style: italic;'></div>
        <div class='comment_content'></div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h2>Re: (Score:2)</h2>
          <div class='by' style='font-style: italic;'>by omnichad ( 1198475 )</div>
          <div class='comment_content'><p></p><p>So you like the way I word it (No Giggity).</p></div>
        </div>
      </div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:3)</h2>
      <div class='by' style='font-style: italic;'>by Solandri ( 704621 )</div>
      <div class='comment_content'><p></p><p>The problem is there are three ways to tweak the DPI. Compatibility Mode, which can be set on a per-app basis. Display scaling, which is set via the Windows desktop and affects everything. And Advanced Scaling which Microsoft introduced with the Spring Creators Update, which I haven't figured out exactly how it's different from the regular Display Scaling.</p><p></p><p>Under the hood, there's also ClearType which runs on top of DPI scaling, and does [1]subpixel rendering  [grc.com] (using the individual red, green, and blue subp</p><p></p><p></p><p></p><p></p><p>[1] https://www.grc.com/ctwhat.htm</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h2>Re: (Score:2)</h2>
        <div class='by' style='font-style: italic;'>by omnichad ( 1198475 )</div>
        <div class='comment_content'><p></p><p>Looking at some zoomed in screenshots, I think they're scaling the subpixel rendering itself (which should never happen). There is no reason to have red or blue tint on anything but the outermost pixel, but looking at my screenshot close up I'm seeing two side-by-side bluish or reddish pixels.</p></div>
      </div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:2)</h2>
      <div class='by' style='font-style: italic;'>by Scoth ( 879800 )</div>
      <div class='comment_content'><p></p><p>Even on Chrome 69 on my multi-DPI work setup I have to turn on the DPI scaling compatibility mode/override to not get huge dialog boxes and widgets. It's a bit frustrating to me that a currently supported, modern app would still have problems with that.</p></div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:2)</h2>
      <div class='by' style='font-style: italic;'>by Ingenium13 ( 162116 )</div>
      <div class='comment_content'><p></p><p>Nope, I have a Windows VM that's essentially stock that I use for a couple applications. I had Chrome open in it earlier today, and noticed that the fonts were blurry and were giving me a headache after a few minutes of use. I thought maybe it was something with RDP messing it up, but after I read this, I logged in again and verified that the fuzzy fonts are only in Chrome. All other applications are fine.</p></div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:2)</h2>
      <div class='by' style='font-style: italic;'>by greenwow ( 3635575 )</div>
      <div class='comment_content'><p></p><p>And most Windows apps. For most of our users on Windows 10, Windows apps have blurry text no matter what settings we try. That really sucks for developers that need to read a lot of text.</p></div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2></h2>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h2>Re: I bet you (Score:2)</h2>
        <div class='by' style='font-style: italic;'>by spongman ( 182339 )</div>
        <div class='comment_content'><p></p><p>They don't?</p></div>
      </div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>Not new (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by Tailhook ( 98486 )</div>
    <div class='comment_content'><p></p><p>Chrome has been having this problem for certain desktop users for a while now. There are workaround flags and whatnot. Just another chronic quality problem in Chrome that never gets fixed....</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:1)</h2>
      <div class='by' style='font-style: italic;'>by MidSpeck ( 1516577 )</div>
      <div class='comment_content'><p></p><p>True. I had to disable "Accelerated 2D canvas" in order to get the fuzziness to go away on mine on Chrome 68, which was working just fine before that.</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2></h2>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:2)</h2>
      <div class='by' style='font-style: italic;'>by Z34107 ( 925136 )</div>
      <div class='comment_content'><p></p><p>Text wrapping is a "solved" problem, too, but you're still doing it by hand for some reason.</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>While they're at it (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by Pedestrianwolf ( 1591767 )</div>
    <div class='comment_content'><p></p><p>..maybe they could also roll back all rounded rectangles they added in v69. Everything is so round it feels like I designed it in my basement.</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:3)</h2>
      <div class='by' style='font-style: italic;'>by ArchieBunker ( 132337 )</div>
      <div class='comment_content'><p></p><p>Chrome has a major UI re-design? Firefox is going to be pulling some late nights to catch up!</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>It's also has performance regressions (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by Suren Enfiajyan ( 4600031 )</div>
    <div class='comment_content'><p></p><p>On my Lenovo G570 the latest chrome also has very high CPU usage on [1]slither.io game [slither.io] and the game turns into a slideshow in places with many snakes. Firefox 61 doesn't suffer from the this problem, despite the fact that its slower almost in any regard. But not this time.</p><p></p><p></p><p></p><p></p><p>[1] http://slither.io/</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>BSOD (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by jimbrooking ( 1909170 )</div>
    <div class='comment_content'><p></p><p>After installing ALL pages I visit are pure black. No text or graphics.</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h2>Re: (Score:2)</h2>
      <div class='by' style='font-style: italic;'>by jimbrooking ( 1909170 )</div>
      <div class='comment_content'><p></p><p>Fix (Win 7 Home): Uninstall Chrome, delete all browsing date. reinstall Chrome, be dazzled by the curviness.</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>Can someone summarize the linked article? (Score:4, Funny)</h2>
    <div class='by' style='font-style: italic;'>by kaizendojo ( 956951 )</div>
    <div class='comment_content'><p></p><p>I upgraded and I can't read it.</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>See Jeff Probst ... (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by CaptainDork ( 3678879 )</div>
    <div class='comment_content'><p></p><p>... about this.</p><p>I remember on Survivor®, back in the day, they had a "blurry tit," problem.</p><p>They fixed that by disallowing exposed tits.</p><p>Google should remove the tits from their fonts.</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>trivial subdomain hiding (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by Zaiff Urgulbunger ( 591514 )</div>
    <div class='comment_content'><p></p><p>So this "trivial subdomain hiding" thing... it's a crap idea right?</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h2>Here's my take as a Mac user (Score:2)</h2>
    <div class='by' style='font-style: italic;'>by DontBeAMoran ( 4843879 )</div>
    <div class='comment_content'><p></p><p>First of all, I really hate these thin fonts. They may look neat on hi-DPI displays but not everyone has one.</p><p>Secondly, the text on the left screen capture looks like it had its pixels hammered to the nearest pixel, which is the typical crappy-looking Microsoft anti-aliasing while the text on the right looks like normal anti-aliased text to me.</p></div>
  </div>
</div>
</body>
